tests/topotests/zebra_rib/test_zebra_rib.py

```diff
@@ -203,7 +203,16 @@ def test_route_map_usage():
     r1.vtysh_cmd("conf\nip route 10.100.100.100/32 192.168.216.3")
     r1.vtysh_cmd("conf\nip route 10.100.100.101/32 10.0.0.44")
     r1.vtysh_cmd("sharp install route 10.0.0.0 nexthop 192.168.216.3 500")
-    sleep(4)
+
+    def check_routes_installed(router):
```

Comment:
The same function name is already in https://github.com/FRRouting/frr/blob/0fae1b83d5a3216fe94abdbeb55d98c0b0c707a9/tests/topotests/zebra_rib/test_zebra_rib.py#L273.

Reply:
fixed